### PR TITLE
Update README.rst

### LINKED ISSUE #185: Update out-of-date README
The README is out-of-date. Needs to be reviewed and edited.

### PR DESCRIPTION
**Pull request by itziakos**
fixes #185

### CODE REVIEW

**Comment by dmsurti** [comment]
The github repo link on Line 5 `http://github.enthought.com/mayavi/mayavi` is incorrect. This also needs to be corrected to ``http://github.com/enthought/mayavi`.

**Comment by dmsurti** [comment]
`Mayavi2_` on Line 7 should just be `Mayavi2' I think.

File: README.rst [hunk 1]
```diff
@@ -76,25 +76,33 @@ examples in the examples directory.
...
 
-http://github.enthought.com/mayavi/mayavi
+Documentation
+==============
 
-or in `docs` directory of the sources.  This includes a man page for the
-`mayavi2` application, a users guide in HTML and PDF format and
-documentation for `mlab`.
+More documentation is available in the `online user manual 
+<http://docs.enthought.com/mayavi/mayavi/>_` or in `docs` directory
```
Comment (dmsurti) [inline README.rst]: This link `<http://docs.enthought.com/mayavi/mayavi/>_` causes a `404` due to the extraneous  `>`.

File: README.rst [hunk 2]
```diff
@@ -108,28 +116,25 @@ also contains some sample data.
...
- https://svn.enthought.com/enthought/
-
-To submit a bug you will necessarily have to register at the site.  Click on
-the "register" link at the top right on the above page to register.  Or login
-if you already have registered.  Once you are registered you may file a bug by
-creating a new ticket.
+The bug tracker is available in github::
 
-Alternatively, you can post on the enthought-dev@mail.enthought.com mailing list.
+https://github.com/enthought/mayavi
```
Comment (dmsurti) [inline README.rst]: Should this be `https://github.com/enthought/mayavi/issues`?

**Comment by itziakos** [comment]
@dmsurti, I have updated the PR. Please have another look.

**Pull request merged**